extensions/markdown-basics/language-configuration.json

```diff
@@ -39,7 +39,7 @@
 			"open": "<",
 			"close": ">",
 			"notIn": [
-				"string"
+				"string", "latex"
```

Comment:
`notIn` only supports a limited set of scopes such as `string`. `latex` won't work